<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDiffVatAmazonUe</class>
 <widget class="QDialog" name="DialogDiffVatAmazonUe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Différence entre les calculs amazon et ceux du logiciel</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonExport">
       <property name="toolTip">
        <string>Exporter au format CSV</string>
       </property>
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDiff">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelTotal">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>différences trouvées</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditOrderId">
       <property name="placeholderText">
        <string>Numéro de commande</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioAll">
       <property name="text">
        <string>Tout</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioNormal">
       <property name="text">
        <string>Normal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioOss">
       <property name="text">
        <string>OSS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioIoss">
       <property name="text">
        <string>IOSS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFilter">
       <property name="toolTip">
        <string>Filtrer par numéro de commande</string>
       </property>
       <property name="text">
        <string>Filtrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonResetFilter">
       <property name="toolTip">
        <string>Annuler le flitre</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewDiff">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogDiffVatAmazonUe</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogDiffVatAmazonUe</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
